### DISCUSSION TITLE
wix 4 - Heat harvesting how to Include/exclude file types and sub folders

### DISCUSSION THREAD

**Discussion by mtissington** [wixtoolset/issues#7615] [Questions]
I'm in the process of upgrading a v3 project and a few years back I used a tool called paraffin to create a wix file each time I compiled the project.

Is there an equivalent way to do this win Wix 4?

After some digging I found this [heat harvesting](https://wixtoolset.org/docs/tools/heat/)

Now my question becomes how do I exclude some sub folders and include/exclude certain file types?

**Comment by BMurri** [comment]
The more performant way involves `Transforms`. The easier to manage way (when using MSBuild) is `HarvestFile`. It's a good excuse to improve your msbuild skills!
